README.md

```diff
@@ -640,6 +641,81 @@ $loginProfile = $osLoginServiceClient->getLoginProfile($formattedName);
 $ composer require google/cloud-oslogin
 ```
 
+## Google Cloud Tasks (Beta)
+- [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/latest/tasks/cloudtasksclient)
+- [Official Documentation](https://cloud.google.com/cloud-tasks/docs)
+
+#### Preview
+
+```php
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Google\Cloud\Tasks\V2beta2\CloudTasksClient;
+use Google\Cloud\Tasks\V2beta2\LeaseDuration;
+use Google\Cloud\Tasks\V2beta2\PullMessage;
+use Google\Cloud\Tasks\V2beta2\PullTarget;
+use Google\Cloud\Tasks\V2beta2\Queue;
+use Google\Cloud\Tasks\V2beta2\Task;
+use Google\Cloud\Tasks\V2beta2\Task_View;
+use Google\Protobuf\Duration;
+
+$client = new CloudTasksClient();
+
+$project = 'example-project';
+$location = 'us-central1';
+$queue = 'example-queue-' . uniqud();
```

Comment:
`uniqid()` - If you'd prefer you can also pass the prefix `example-queue-` in as the first argument of the function for the same effect.

Also, considering the issue in the VI sample last week, we probably want to set something up to make sure we are snippet testing our READMEs just like our clients. @jdpedrie do you think our snippet parsing framework could support this?

Reply:
I've been thinking the same thing regarding README testing. I'll set some time aside and dig into it soon.